reformat-source.sh

```diff
@@ -0,0 +1,4 @@
+#! /bin/sh
+
+black -l 100 odxtools examples tests
```

Comment:
keep black config in pyproject.toml instead of cli arguments, so that IDE takes it into account.

Reply:
ok. I switched the code formatter to `YAPF` because I like its result better than those of `black`. Besides having a *huge* number of knobs, yapf seems to prefer to be configured  via `pyproject.toml`...